code/modules/surgery/organs/pain.dm

```diff
@@ -1,62 +1,47 @@
-mob/var/list/pain_stored = list()
-mob/var/last_pain_message = ""
-mob/var/next_pain_time = 0
+/mob/living/carbon/human
+	var/last_pain_message = ""
+	var/next_pain_time = 0
 
 // partname is the name of a body part
 // amount is a num from 1 to 100
-mob/living/carbon/proc/pain(var/partname, var/amount, var/force, var/burning = 0)
-	if(stat >= 2) return
+/mob/living/carbon/human/proc/pain(partname, amount)
+	if(stat >= UNCONSCIOUS)
+		return
 	if(reagents.has_reagent("sal_acid"))
 		return
 	if(reagents.has_reagent("morphine"))
 		return
 	if(reagents.has_reagent("hydrocodone"))
 		return
-	if(world.time < next_pain_time && !force)
+	if(world.time < next_pain_time)
 		return
-	if(amount > 10 && istype(src,/mob/living/carbon/human))
-		if(paralysis)
-			AdjustParalysis(-round(amount/10))
-	if(amount > 50 && prob(amount / 5))
-		src:drop_item()
 	var/msg
-	if(burning)
-		switch(amount)
-			if(1 to 10)
-				msg = "<span class='danger'>Your [partname] burns.</span>"
-			if(11 to 90)
-				msg = "<span class='danger'><font size=2>Your [partname] burns badly!</font></span>"
-			if(91 to 10000)
-				msg = "<span class='danger'><font size=3>OH GOD! Your [partname] is on fire!</font></span>"
-	else
-		switch(amount)
-			if(1 to 10)
-				msg = "<b>Your [partname] hurts.</b>"
-			if(11 to 90)
-				msg = "<b><font size=2>Your [partname] hurts badly.</font></b>"
-			if(91 to 10000)
-				msg = "<b><font size=3>OH GOD! Your [partname] is hurting terribly!</font></b>"
+	switch(amount)
+		if(1 to 10)
+			msg = "<b>Your [partname] hurts.</b>"
+		if(11 to 90)
+			msg = "<b><font size=2>Your [partname] hurts badly.</font></b>"
+		if(91 to INFINITY)
+			msg = "<b><font size=3>OH GOD! Your [partname] is hurting terribly!</font></b>"
```

Comment:
Would it not be better to use `<span>` classes for these warnings?

Reply:
Warning and danger specifically, but what's the big ass danger one? If you stubbed your toe on a supermatter shard and took 101 damage the message should probably be made super obv.

Reply:
"You clumsy oaf"